site/controller/UserController.php

```diff
@@ -1,4 +1,56 @@
 <?php
 
-    // var_dump($_FILES);
-    // var_dump($_POST);
+require '../database/ConnectDB.php';
```

Comment:
Cái này a có thể return ở trong file ConnectDB.php xong ở đây thì làm như này sẽ chủ động hơn thay vì dùng 1 biến global duy nhất

```suggestion
$connect = require '../database/ConnectDB.php';
```